import UserTas1_pb2

funcode_dict = {
    131293: UserTas1_pb2.LoginReq,
    131294: UserTas1_pb2.LoginRsp,
    131295: UserTas1_pb2.LogoutReq,
    131296: UserTas1_pb2.LogoutRsp,
    131327: UserTas1_pb2.CustOfflineNtf,
    131329: UserTas1_pb2.MemberLoginReq,
    131330: UserTas1_pb2.MemberLoginRsp,
    131331: UserTas1_pb2.ErrorNtf,
    131632: UserTas1_pb2.P2LStatusSync,
    132205: UserTas1_pb2.TokenCheckReq,
    132206: UserTas1_pb2.TokenCheckRsp,
    16908333: UserTas1_pb2.QueryOnlineUserReq,
    16908334: UserTas1_pb2.QueryOnlineUserRsp,
    }
